pwa/src/templates/mappingDetailTemplate/MappingDetailsTemplate.tsx

```diff
@@ -51,7 +97,65 @@ export const MappingDetailTemplate: React.FC<MappingDetailsTemplateProps> = ({ m
             showTitleTooltip
           />
 
-          <MappingFormTemplate mapping={getMapping.data} />
+          <div className={styles.tabContainer}>
+            <TabContext value={currentTabs.mappingDetailTabs.toString()}>
+              <Tabs
+                value={currentTabs.mappingDetailTabs}
+                onChange={(_, newValue: number) => {
+                  setCurrentTabs({ ...currentTabs, mappingDetailTabs: newValue });
+                }}
+                variant="scrollable"
+              >
+                <Tab className={styles.tab} label={t("General")} value={0} />
+                <Tab className={styles.tab} label={t("Test Mappings")} value={1} />
+              </Tabs>
+
+              <TabPanel className={styles.tabPanel} value="0">
+                <MappingFormTemplate mapping={getMapping.data} />
+              </TabPanel>
+
+              <TabPanel className={styles.tabPanel} value="1">
+                <form onSubmit={handleSubmit(onSubmitTest)}>
+                  <Button
+                    label={t("Test mapping")}
+                    icon={faArrowsRotate}
+                    variant="primary"
+                    className={clsx(styles.buttonIcon, styles.testMappingButton)}
+                    disabled={isLoading.mappingForm}
+                    type="submit"
+                  ></Button>
```

Comment:
Because we're adding a `className` here, the `className` _within_ the `Button` component is overwritten (which is an unfortunate side effect, we should look into sometime..).

However: right now, we're adding one class `.buttonIcon` which is adding styling that is _already_ in the `Button` component itself.

And we're adding the `.testMappingButton` class which is rendering margin (position in context).

The solution here would be to _remove_ both classes, this way the margin (positioning) will be lost, but the `.buttonIcon` logic will still be there, because the styling from `Button` will no longer be overwritten.

The margin can be added by the container where this `<Button />` is implemented, in this case the `<form>`, like so:

`<form className={...} ...>`

```
.form > *:not(:last-child) {
 margin-block-end: ...
}
```